src/app/modules/main/chat_section/chat_content/messages/view.nim

```diff
@@ -128,6 +128,11 @@ QtObject:
   proc setEditModeOn*(self: View, messageId: string) {.slot.} =
    self.model.setEditModeOn(messageId)
 
+  proc setEditModeOnLastMessage*(self: View, pubkey: string) {.slot.} =
+    let lastMessage = self.model.getLastItem()
```

Comment:
This should be `getOutLastMessage`. We don't want to edit someone else's message (even as an admin)

Reply:
What the heck where did the check go.
This was already implemented. Lemme update this PR